Code/GraphMol/Wrap/rough_test.py

```diff
@@ -8628,6 +8628,13 @@ def testGithub8877(self):
     self.assertRaises(ValueError, lambda: m.GetBondWithIdx(0).SetQuery(None))
     self.assertRaises(ValueError, lambda: m.GetBondWithIdx(0).ExpandQuery(None))
 
+  def testBondChiralityInversion(self):
+    mol1 = Chem.MolFromSmiles("Cc1cccc(F)c1-c1c(C)cccc1Cl |wU:7.7,&1:7|")
+    mol2 = Chem.MolFromSmiles("Cc1cccc(F)c1-c1c(C)cccc1Cl |wU:7.6,&1:7|")
+    self.assertNotEquals(mol1.GetBonds()[7].GetStereo(), mol2.GetBonds()[7].GetStereo())
+    mol1.GetBonds()[7].InvertChirality()
+    self.assertEquals(mol1.GetBonds()[7].GetStereo(), mol2.GetBonds()[7].GetStereo())
```

Comment:
`assertEquals` was removed in python 3.12:
```suggestion
    self.assertEqual(mol1.GetBonds()[7].GetStereo(), mol2.GetBonds()[7].GetStereo())
```